assets/js/app/Data.js

```diff
@@ -281,3 +288,43 @@ LocusZoom.KnownDataSources = [
     LocusZoom.Data.LDSource,
     LocusZoom.Data.GeneSource];
 
+
+// This function expects a string of transformations
+// in the form "|name1|name2" and returns a proper
+// js function to perform the transformation
+LocusZoom.Data.getTransformation = function(x) {
+    if (x) {
+        var getfun = function(x) {
+            var fun = LocusZoom.Data.KnownTransformations[x];
+            if (fun)  {
+                return fun;
+            } else {
+                throw("transformation " + x + " not found");
+            }
+        };
+        var funs = [];
+        var fun;
+        var re = /\|([^\|]+)/g;
+        var result;
+        while((result = re.exec(x))!=null) {
+            funs.push(result[1]);
+        }
+        if (funs.length==1) {
+            return getfun(funs[0]);
+        } else if (funs.length > 1) {
+            return function(x) {
+                var val = x;
+                for(var i = 0; i<funs.length; i++) {
+                    val = getfun(funs[i])(val);
+                }
+                return val;
+            };
+        }
+    } 
+    return null;    
+};
+
+// This of known transformations (thus it can be extended)
+LocusZoom.Data.KnownTransformations = {
```

Comment:
First impression is that this is a nice intuitive pattern for such functions. "Known" may not be necessary in the name since they're just Transformations and if one isn't found/known we'll be [throwing an exception anyway](https://github.com/statgen/locuszoom/pull/29/files#diff-a6139e2504656ffc08c847116c79ce29R302).

We might also want to consider adding a `LocusZoom.Data.addKnownTransformation` (or just `addTransformation`) method to let users define their own too. I can definitely see how Transformations in general would be a powerful feature!